src/resmom/mom_hook_func.c

```diff
@@ -3142,7 +3151,15 @@ record_job_last_hook_executed(unsigned int hook_event,
 	char chr_save = '\0';
 	char *p_dir = NULL;
 
-	if ((hook_name == NULL) || (pjob == NULL)) {
+	if (pjob == NULL) {
+		log_event(PBSEVENT_DEBUG2, PBS_EVENTCLASS_HOOK,
+			  LOG_INFO, "", "Job not received");
+		return;
+	} 
+
+	if (hook_name == NULL) {
+		log_event(PBSEVENT_DEBUG2, PBS_EVENTCLASS_HOOK,
+			  LOG_INFO, "", "Hook not received");
 		return;
 	}
 	if (hook_event != HOOK_EVENT_EXECJOB_PROLOGUE) {
```

Comment:
line 3169: 
	if (filepath != NULL) {
log error if condition when filepath==NULL

Reply:
Done in https://github.com/PBSPro/pbspro/pull/1737/commits/06f24712f1b101a386d5b88656a24489250e52f0